src/lib/components/TitleShowcase/TitleShowcasesContainer.svelte

```diff
@@ -65,11 +65,51 @@
 		}
 	});
 
-	const tmdbPopularMoviesPromise = getTmdbPopularMovies()
-		.then((movies) => Promise.all(movies.map((movie) => getTmdbMovie(movie.id || 0))))
-		.then((movies) => movies.filter((m) => !!m).slice(0, 10));
+	let popularMovies: (
+		| {
```

Comment:
OOI is there any strong opinions about formatting? Tab alignment kills me 😭

Reply:
No strong opinions, I usually go for spaces but I think the project was initialized with tabs 🤷